File: python3/1046.py
inicio, fim = input().split(" ")
inicio = int(inicio)
fim = int(fim)

if(fim > inicio):
    tempo = fim - inicio
elif(inicio > fim):
    tempo = fim + 24 - inicio
else:
    tempo = 24

print("O JOGO DUROU {} HORA(S)".format(tempo))

File: c++/1046.cpp
#include <iostream>

using namespace std;

int main() {

   int a, b;
   cin >> a >> b;

   if (a < b) {
    cout << "O JOGO DUROU " << b - a << " HORAS(S)" << endl;
   }
   else if (a > b) {
    cout << "O JOGO DUROU " << (b + 12) - (a - 12) << " HORAS(S)" << endl;
   }
   else if (a == b) {
    cout << "O JOGO DUROU 24 HORA(S)" << endl;;
   }
   return 0;
}
